CuBoulder/express_mono
# Mathjax Feature/26

To test:

1. Create a basic page
2. add the following content:
`When $a \ne 0$, there are two solutions to \(ax^2 + bx + c = 0\) and they are
$$x = {-b \pm \sqrt{b^2-4ac} \over 2a}.$$`
3. save

![Screen Shot 2019-05-20 at 12 27 23 PM](https://user-images.githubusercontent.com/349713/58043913-b1cea280-7afb-11e9-9cb1-34874e8b1100.png)

The way that this is implemented, so as not to override the wysiwyg feature, you need to test the result, not look to see if it's enabled on the configuration form (it won't be).
